codegen/java/java-entities/src/main/java/com/stanfy/helium/handler/codegen/java/entity/PojoWriter.java

```diff
@@ -69,4 +71,9 @@ public void writeConstructors(final Message message) {
     // nothing
   }
 
+  @Override
```

Comment:
Actually, after this PR, PojoWriter isn't the best name for this class. What do you think, If I introduce some `factory`, which will provide `EnumWriter` and `PojoWriter`?